app/routes/auth.js

```diff
@@ -22,7 +23,14 @@ router.post(
     failureRedirect: '/auth/login',
     failureFlash: 'Brukernavn og/eller passord er feil.',
   }),
-  (req, res) => {
+  async (req, res) => {
+    const { _id, unActivatedEmail } = req.user;
+    // If they can login with an unActivatedEmail user we'll set the user's
+    // unActivatedEmail to blank to show that this user has been able to login
+    if (unActivatedEmail) {
+      await User.findByIdAndUpdate({ _id: _id }, { unActivatedEmail: '' });
```

Comment:
:heart_eyes:

Reply:
just to learn something 💯  I see how both ways work, why is this considered better?